### DISCUSSION TITLE
[Feature Request] : Add value notation for each bar in chart bars

### DISCUSSION THREAD

**Discussion by alamenai** [tremorlabs/tremor-npm#804] [Ideas]
Thank you Tremo team for this amazing project.

In fact, it helps us a lot in the data visualization of our product.

Our users are aged a little bit and I used Tremor to create a page to visualize the data and then allow them to download it as PDF.

However, I would ask if it's possible to annotate the value on the top of each bar.

This is Figma example:

![image](https://github.com/tremorlabs/tremor/assets/20143684/48ac939a-c32f-4c42-9bed-b7ca6eb57cc5)

Anf if it's possible in the current version, please could you provide an example because I'm prototyping based on your UI kit and I don't know if it's possible to do it with code.

Thank you

**Comment by gustavospriebe** [comment]
Any other way to do this while is not implemented? I need to show the value of each bar.

**Reply by alamenai** [reply]
From my side, I switched to [barlist](https://www.tremor.so/docs/visualizations/barlist) and changed the orientation of the card.
